wallet/src/model.rs

```diff
@@ -40,6 +40,15 @@ pub struct Addresses {
     pub total: u32,
 }
 
+#[derive(Debug, Serialize, Deserialize, PartialEq)]
+pub struct AddressInfo {
+    pub label: Option<String>,
```

Comment:
Does it make sense to add the serde skip attribute to the optional fields?

    #[serde(skip_serializing_if = "Option::is_none")]

Reply:
We checked that the `skip_serializing_if` is not working properly with the `bincode` which we use for the DB. :disappointed:  Hopefully, we'll change it in the near future for `protobuf` or similar.